hibernate-core/src/main/java/org/hibernate/boot/model/naming/ObjectNameNormalizer.java

```diff
@@ -51,7 +51,7 @@ public Identifier normalizeIdentifierQuoting(Identifier identifier) {
 
 	/**
 	 * Normalizes the quoting of identifiers.  This form returns a String rather than an Identifier
-	 * to better work with the legacy code in {@link org.hibernate.mapping}
+	 * to better work with the legacy code in {@code org.hibernate.mapping} package.
```

Comment:
There is no possibility to link to package in Javadoc, so I replace `@link` with `@code`. Again, let me know if you have better idea.

Reply:
I think its possible when you add a label to it: `{@link org.hibernate.mapping mapping}

Reply:
I tried it again to no avail. The testing is via command line `./gradlew :hibernate-core:javadoc`. It shows up as warning. However, in Intellij IDE most of the javadoc issues in this PR won't be an issue.